unlock-protocol-com/blog/cerfification-launch.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Oh Yeah? Prove it! Certifications by Unlock
+authorName: Kalidou Diagne
+publishDate: April 11, 2023
+description: Unlock introduces an easy way to issue transparent, provable credentials for professional certifications, continuing education credits, industry expertise, and more.
+image: /images/blog/certifications/certification-cover-min.png
+---
+
+![certification-cover-min.png](/images/blog/certifications/certification-cover-min.png)
+
+Unlock has launched an easy way to issue certifications and credentials on-chain. Every certification held is a proof of a course completion and has its own verifiable NFT.
+
+Let's see how to create a certification or credential and distribute it to participants. 
+
+## Certifications powered by Unlock Protocol
+
+First, visit [https://certifications.unlock-protocol.com](https://certifications.unlock-protocol.com), where you can learn about the benefits of certification and credentialing NFTs powered by Unlock Protocol.
+
+## Enter information about your certification
+
+Click "Create your certification now” to go to the form to build your certification. You’ll put in a few pieces of information to set up your certification.
+
+### Basic certification information
+
+Enter the basic information for your certification including: 
+
+- **Certification name**: The program or course name that was completed by the recipient of the certification.
+- **Description**: The description for the certification. This will be part of the NFT metadata.
+- **Official issuer name**: This is the name of the issuing organization for this certification. This is part of NFT metadata.
+- **External URL**: Include a link to the URL of the organization that performed the certification. This will be included in the NFT metadata.
+- **Image**: The main image logo that will be shown for the on-chain certificate NFT.
+- **Certification duration:** How long the certification will be valid for. Certifications have an unlimited duration by default. This duration can be changed during setup to whatever timeframe is appropriate for your situation.
+
+![basic-info-min.png](/images/blog/certifications/basic-info-min.png)
+
+### Price and quantity
+
+By default, your certification can't be purchased. By default, it can only be granted (”airdropped”) to a recipient. (We’ll explore how to perform the airdrop later in this article.)
```

Comment:
```suggestion
By default, your certification can't be purchased: it can only be granted (”airdropped”) to a recipient. (We’ll explore how to perform the airdrop later in this article.)
```